tests/run_tests.py

```diff
@@ -129,6 +129,8 @@ def _print_skip_message(test, skip_tag):
         reason = "skipping test that requires special configuration"
     elif skip_tag == TestTags.CORE:
         reason = "skipping non-core test"
+    elif skip_tag == TestTags.SOURCEONLY:
+        reason = "skipping test that can run only against library compiled from source"
```

Comment:
I think this commit can be squashed into the previous one.

Reply:
Fixed.